egs/wsj/s5/steps/nnet3/libs/rnn_train_lib.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python
+
+
+# Copyright 2016 Vijayaditya Peddinti.
+#           2016 Vimal Manohar
+# Apache 2.0.
+
+import logging
+import imp
+
+imp.load_source('nnet3_train_lib', 'steps/nnet3/nnet3_train_lib.py')
```

Comment:
Its not importing three times. First is to load module from source, then import module and then import functions from module.